src/views/RecIM/views/Team/index.jsx

```diff
@@ -0,0 +1,194 @@
+import { Grid, Typography, Card, CardHeader, CardContent, Button, IconButton } from '@mui/material';
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router';
+import styles from './Team.module.css';
+import GordonLoader from 'components/Loader';
+import GordonUnauthorized from 'components/GordonUnauthorized';
+import Header from '../../components/Header';
+import TeamForm from 'views/RecIM/components/Forms/TeamForm';
+import InviteParticipantForm from '../../components/Forms/InviteParticipantForm';
+import { useUser } from 'hooks';
+import { ParticipantList, MatchList } from '../../components/List';
+import { getTeamByID } from 'services/recim/team';
+import { getParticipantByUsername } from 'services/recim/participant';
+import AddCircleRoundedIcon from '@mui/icons-material/AddCircleRounded';
+import EditIcon from '@mui/icons-material/Edit';
+
+const Team = () => {
+  const { teamID } = useParams();
+  const { profile } = useUser();
+  const [team, setTeam] = useState();
+  const [loading, setLoading] = useState(true);
+  const [user, setUser] = useState();
+  const [openTeamForm, setOpenTeamForm] = useState(false);
+  const [hasPermissions, setHasPermissions] = useState(false);
+  const [openInviteParticipantForm, setOpenInviteParticipantForm] = useState(false);
+  const handleInviteParticipantForm = (status) => {
+    //if you want to do something with the message make a snackbar function here
+    setOpenInviteParticipantForm(false);
+  };
+  useEffect(() => {
+    const loadTeamData = async () => {
+      setLoading(true);
+      setTeam(await getTeamByID(teamID));
+      if (profile) {
+        setUser(await getParticipantByUsername(profile.AD_Username));
+      }
+      setLoading(false);
+    };
+    loadTeamData();
+  }, [profile, teamID, openTeamForm, openInviteParticipantForm]);
+  // @TODO modify above dependency to only refresh upon form submit (not cancel)
+
+  //checks if the team is modifiable by the current user
+  useEffect(() => {
+    let hasCaptainPermissions = false;
+    if (user) {
+      if (team) {
+        let role =
+          team.Participant.find((teamParticipant) => teamParticipant.Username === user.Username)
+            ?.Role ?? 'Invalid';
```

Comment:
Since `role` is only referenced here, there's no need to provide a default value, just let it be undefined.
```suggestion
        const role =
          team.Participant.find((teamParticipant) => teamParticipant.Username === user.Username)
            ?.Role;
```